apps/src/p5lab/AnimationTab/new-list-item.module.scss

```diff
@@ -0,0 +1,65 @@
+@import 'color.scss';
+
+$polar-blue: #0094CA;
+
+.tile {
+  width: 95%;
+  padding: 0 0 4px 0;
+  margin-bottom: 0;
+  background: none;
+  border: 20px;
+  margin-right: 0;
+  margin-left: 5px;
+
+  &:hover {
+    cursor: pointer;
+    box-shadow: none;
+
+    .border {
+      background-color: $polar-blue;
+      border: solid 4px $polar-blue;
+    }
+
+    .addButton {
+      color: $white
```

Comment:
Nit: semicolon. Also, do we like newlines at end of these files?

Reply:
Good catch! It looks like our linter should catch that https://github.com/stylelint/stylelint-config-standard/blob/main/index.js#L66 –– it looks like the prettier config conflicts. I took that out and fixed the file with the linter, which fixes the semicolon! Will bring this to the larger group.

As for newlines, it looks like the linter complains about >1 new line, but 0 or 1 newlines both pass (after blocks as well). I've added it for consistency, but that might be another linter candidate?